CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v1.0.1 Changes
```

Comment:
Typically the package maintainer(s) handle release version bumps, so in this case I'd suggest changing from `v1.0.1` to `unreleased` and revert your metadata.rb version bump